analysis/analysis.py

```
@@ -0,0 +1,135 @@
+
+            with open(filepath, 'r', encoding='utf-8') as f:
+                try:
+                    data = json.load(f)
+                except json.JSONDecodeError:
+                    print(f"Ошибка чтения JSON файла: {filepath}")
+                    continue
+
+                # Извлекаем метрики
+                metrics = data.get('metrics', {})
+
+                for question_id, metric_list in metrics.items():
```

Ruff (0.15.13)

[warning] Loop control variable `question_id` not used within loop body

Rename unused `question_id` to `_question_id`

(B007)